frontends/election-manager/src/screens/ballot_list_screen.tsx

```diff
@@ -20,12 +20,15 @@ import { ExportBallotPdfsButton } from '../components/export_ballot_pdfs_button'
 
 const Header = styled.div`
   display: flex;
-  flex-direction: row-reverse;
```

Comment:
@arsalansufi what does this change do?

Reply:
So this `flex-direction: row-reverse` was making it such that the children of the `Header` were rendered in the reverse of the order in which they were declared. To get the Precinct/Style toggle to show up to the left of the export buttons, we accordingly had to declare the export buttons first.

That's fine! But a little unintuitive. I just got rid of the `flex-direction: row-reverse` and declared the Precinct/Style toggle first.

`flex-direction: row-reverse` makes more sense when you're designing responsive layouts, and you, for example, want items to appear in a completely different layout / arrangement on mobile vs. desktop.